### DISCUSSION TITLE
v1.0.0-beta8

### DISCUSSION THREAD

**Discussion by Mrs4s** [Mrs4s/go-cqhttp#1147] [General]
> **注意: 此版本为开发者前瞻测试版, 包含大量修改, 可能存在不可预见的错误, 请谨慎更新**

新增: 在输入时将加密Key隐藏. by @fumiama 
新增: 支持使用多个数据库持久化储存. by @Mrs4s 
新增: `MongoDB` 数据库支持. by @Mrs4s 
新增: `btree` 数据库支持. by @wdvxdr1123 
新增: 支持通过 `btree` 储存缓存信息. by @wdvxdr1123 
新增: 支持控制台染色输出. by @fzls 
新增: 支持模块化编译功能. by @wdvxdr1123 
新增: 支持网络状态诊断. by @Mrs4s 
新增: 支持解析股子消息 `[CQ:dice]`. by @Mrs4s 
新增: 支持解析闪照链接. by @Mrs4s 
**新增: 支持QQ频道.** [文档](https://github.com/Mrs4s/go-cqhttp/blob/master/docs/guild.md) by @Mrs4s 

修复: 在未配置 `password` 项目时, Token失效后启动会导致预料之外登录错误的问题. by @Mrs4s 
修复: 在部分平台上由于 `silk` 模块无法编译通过的问题. by @wdvxdr1123 
修复: 单独回复的消息ID解析错误的问题. by @wdvxdr1123 
修复: 在极为罕见的情况下, 解析畸形数据包时可能会崩溃的问题. by @Mrs4s 

优化: `Android Phone` 协议升级到了 `v8.8.38` by @Mrs4s 
优化: `iPad` 协议升级到了 `v8.8.35` by @Mrs4s 
优化: 对代码结构进行了整体优化. by @wdvxdr1123 
优化: 对数据包解析逻辑进行了优化. by @LXY1226
优化: 对数据包解码速度进行了优化. by @wdvxdr1123 
优化: 缩小了登录时二维码的打印大小. by @wdvxdr1123 
优化: 对API路由的生成进行了自动化. by @wdvxdr1123 

<hr /><em>This discussion was created from the release <a href='https://github.com/Mrs4s/go-cqhttp/releases/tag/v1.0.0-beta8'>v1.0.0-beta8</a>.</em>

**Comment by 18844626311** [comment]
这个版本post上报器好像有点问题

**Comment by Miuzarte** [comment]
更新之后ws连不上
更新日志里(好像)没看到跟ws有关的条目
![image](https://user-images.githubusercontent.com/66856838/141668590-27178a54-2267-40af-9814-428e3890ef17.png)

**Reply by Ink-33** [reply]
bug，修了，等发版
